### PR TITLE
[filecmd] update to 5.33

### PR DESCRIPTION
**Pull request by lovedheart**

### CODE REVIEW

File: packages/filecmd.rb [hunk 1]
```diff
@@ -3,33 +3,22 @@
...
+  version '5.33'
+  source_url 'ftp://ftp.astron.com/pub/file/file-5.33.tar.gz'
+  source_sha256 '1c52c8c3d271cd898d5511c36a68059cda94036111ab293f01f83c3525b737c6'
 
   def self.build
-    system "./configure --libdir=#{CREW_LIB_PREFIX}"
+    system "./configure",
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}",
+           "--disable-libseccomp"
```
Comment (uberhacker) [inline packages/filecmd.rb]: What is the reason to exclude libseccomp support?

Comment (lovedheart) [reply packages/filecmd.rb]: No libseccomp available. Cant pass configuration

Comment (uberhacker) [reply packages/filecmd.rb]: Add new package?  See https://github.com/seccomp/libseccomp.

Comment (lovedheart) [reply packages/filecmd.rb]: Maybe merge this first. This is not critical

Comment (uberhacker) [reply packages/filecmd.rb]: Just curious why we need to disable now and not with earlier versions.

Comment (lovedheart) [reply packages/filecmd.rb]: It can not automatically detect the existance of libseccomp any more.

**Pull request merged**